Add code resources page (#1027)
BitcoinDesign/Guide: guide/resources/code-resources.md

---
layout: guide
title: Code resources

> description: Bitcoin Design Guide code resources.

Reviewer: ```suggestion
description: Code implementations of reference designs and other resources from the Bitcoin Design Guide, created by the community.
```
Making the description more elaborate.

nav_order: 4
parent: Resources
permalink: /guide/resources/code-resources/
main_classes: -no-top-padding
---

<!--

Editor's notes

Illustration source:

-->

# Code resources

Although mostly focused on design, some members of the community also have their hands in various code resources that are based on, or influenced by the Bitcoin Design Guide. They are all open source and free to use, modify and contribute to.

## Bitcoin UI Kit packages

> The Bitcoin UI Kit is a great resource for designing bitcoin products. While the main effort is on the design components and Figma files, there are a couple of code implementations by community memebers that make adopting the designs in the guide much easier.

Reviewer: ```suggestion
The [Bitcoin UI Kit](https://www.bitcoinuikit.com) is a great resource for designing bitcoin products. While the main effort is on the design components and design source files, there are a couple of code implementations by community members that make adopting the designs in the guide much easier.
```
Adding a link for convenience, type, and removing Figma to be more general (ahem, Penpot, ahem).

- Bitcoin UI for [Swift](https://github.com/reez/WalletUI)

> - Bitcoin UI for [Flutter](https://pub.dev/packages/bitcoin_ui_kit)

Reviewer: This links to a page that says "discontinued".

Author: Fixed (this stemmed from changing the name of the package)

- Bitcoin UI for [React Native](https://github.com/EttaWallet/etta-ui) (Etta UI)

## Wallet implementations

Several different projects are building on the reference designs from the guide. They all have slightly different goals and are at different stages, but are all fantastic learning resources for other people wanting to build a wallet.

### Etta wallet

[Etta Wallet](https://github.com/EttaWallet/EttaWallet) is a resource for experimenting with non-custodial lightning with a strong bias towards usability, accessibility and good UX. Built with React Native.

> The project is strongly inspired by the [daily spending wallet]('/guide/daily-spending-wallet/').

Reviewer: ```suggestion
The project is strongly inspired by the [daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }}).
```
